pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -98,6 +98,7 @@ func (c *Cluster) makeConfigInitContainer(mgrConfig *mgrConfig) v1.Container {
 					}}},
 			k8sutil.PodIPEnvVar(k8sutil.PrivateIPEnvVar),
 			k8sutil.PodIPEnvVar(k8sutil.PublicIPEnvVar),
+			k8sutil.PodIPEnvVar("ROOK_MGR_MODULE_SERVER_ADDR"),
```

Comment:
We have two other env vars that have the Pod IP in them (see line 99-100) do we really need a new env var here?

Reply:
while it's not strictly necessary, i wanted to be clear about the purpose of this setting. If the public and private ip configuration changes in the future, we should reevaluate this setting as well. This way it will be more obvious to evaluate it.